resources/builder/builder-role-binding-core.yaml

```diff
@@ -18,13 +18,13 @@
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: camel-k-builder
+  name: camel-k-builder-core
```

Comment:
Could this be called `camel-k-builder` to be symmetrical with the operator Role and RoleBinding? Also have the file renamed to `builder-role-binding` for the same reason?